src/com/facebook/buck/util/concurrent/BUCK.autodeps

```diff
@@ -9,6 +9,7 @@
       "//src/com/facebook/buck/util/immutables:immutables",
       "//third-party/java/concurrent-locks:concurrent-locks",
       "//third-party/java/guava:guava",
+      "//third-party/java/errorprone:error-prone-annotations",
```

Comment:
I think the problm is even more subtle here: There is no mentions of error-prone-annotations annotation in Buck code. It's the guava that indirectly depends on it, and crashing when it's not provided. So it would need to be investigated. Anyway, changing:

```
diff --git a/.buckconfig b/.buckconfig
index b47016f..05da64d 100644
--- a/.buckconfig
+++ b/.buckconfig
@@ -14,7 +14,7 @@
     maven-importer = //src/com/facebook/buck/maven:resolver
 [autodeps]
    include_signature = false
-   java-package-mappings = com.google.caliper => //third-party/java/caliper:caliper
+   java-package-mappings = com.google.caliper => //third-party/java/caliper:caliper,com.google.errorprone.annotations => //third-party/java/errorprone:error-prone-annotations
 [buildfile]
     includes = //DEFS
 [log]
```

and running `buck autodeps` havn't added eror-prone to autodeps.